values.yaml

```
@@ -7,6 +7,12 @@ agent:
     repository: jetbrains/teamcity-agent
     tag: latest
     pullPolicy: Always
+
+  ## Specify an array of imagePullSecrets.
+  ## Secrets must be manually created in the namespace.
+  ## ref: https://kubernetes.io/docs/concepts/containers/images/#specifying-imagepullsecrets-on-a-pod
+  imagePullSecrets:
```

here and elsewhere: I think the correct way of doing this is:
```suggestion
  imagePullSecrets: []
```